frontend/src/components/widgets/ActionLogCards.js

```diff
@@ -35,14 +35,12 @@ const useStyles = makeStyles((theme) => ({
     variant: "outlined",
     color: "default",
     size: "small",
-    ["@media (max-width:620px)"]: {
-      // eslint-disable-line no-useless-computed-key
+    "@media (max-width:620px)": {
```

Comment:
Why did you remove the comments? Do you think the es-lint warnings should be there?  Sometimes it is good to suppress certain warnings

Reply:
I removed the square brackets and that resolved the ES Lint warnings. https://eslint.org/docs/latest/rules/no-useless-computed-key

So I did not think the comments were still nessesary.